### PR TITLE
ENH: include nvidia-smi across lectures

### LINKED ISSUE #51: [hardware] Include nvidia-smi output in lectures
@jstac here is a preview of the `nvidia-smi` output

https://6458e26e627cb621dbb3499a--incomparable-parfait-2417f8.netlify.app/wealth_dynamics.html

Let me know if you would like to role this out more widely.

### PR DESCRIPTION
**Pull request by mmcky**
fixes #51

### CODE REVIEW

**Comment by mmcky** [comment]
The linkchecker is failing due to the anchor issue #53 so this looks ready.

**Comment by mmcky** [comment]
@jstac the only lecture I didn't add `nvidia-smi` to is

https://6462d2eada9c00149ee7641a--incomparable-parfait-2417f8.netlify.app/jax_intro.html

thinking it simplifies the lecture. But we can add it there as well if you like.

**Comment by chappiewuzefan** [comment]
Hi @mmcky Matt , I think the left broken link was fixed by @Smit-create in [53e0674](https://github.com/QuantEcon/lecture-jax/pull/62/commits/53e06746554dbaa65544e7be55c97def4740273c)

File: lectures/aiyagari_jax.md [hunk 1]
```diff
@@ -57,6 +57,12 @@ import jax
 import jax.numpy as jnp
 ```
 
+We can also check the `gpu` hardware we are running on using `nvidia-smi`.
```
Comment (jstac) [inline lectures/aiyagari_jax.md:60]: Thanks @mmcky 

How about "Let's check the GPU we are running" 

I'm very keen on cutting words ATM.  Either way `gpu` should be GPU.

Comment (mmcky) [reply lectures/aiyagari_jax.md]: @jstac done.

**Comment by jstac** [comment]
Thanks @mmcky !

**Pull request merged**